### PR TITLE
Add the new tower and tpv to the robot description

### PR DESCRIPTION
**Pull request by benmalenfant**
image

### CODE REVIEW

**Comment by benmalenfant** [comment]
need to add zed_wrapper as dependency and doccument

**Comment by Leuchak** [comment]
There are extra spaces in markhor.urdf.xacro and tower.xacro after removing them, it'll be good.

**Comment by benmalenfant** [comment]
> There are extra spaces in markhor.urdf.xacro and tower.xacro after removing them, it'll be good.

Could you point out where and i'll fix it

**Review by McCaroon — DISMISSED** [review]

File: markhor_description/urdf/tower.xacro [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            </geometry>
+            <origin xyz="0 0 0" rpy="${pi/2} 0 ${pi/2}" />
+        </collision>
+        <xacro:markhor_inertial type="body" />
+    </link>
+
+    <joint name="tpv_base_j" type="continuous">
+        <parent link="tower_top" />
+        <child link="tpv_base" />
+        <axis xyz="0 0 1"/>
```
Comment (Leuchak) [inline markhor_description/urdf/tower.xacro:82]:
```suggestion
        <axis xyz="0 0 1"/>
```

File: markhor_description/urdf/tower.xacro [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            </geometry>
+            <origin xyz="0 0 0" rpy="${pi/2} 0 ${pi/2}" />
+        </collision>
+        <xacro:markhor_inertial type="body" />
+    </link>
+
+    <joint name="tpv_tilt_j" type="continuous">
+        <parent link="tpv_tilt_axis" />
+        <child link="tpv_tilt" />
+        <axis xyz="0 1 0"/>
```
Comment (Leuchak) [inline markhor_description/urdf/tower.xacro:123]:
```suggestion
        <axis xyz="0 1 0"/>
```

**Review by Leuchak — COMMENTED** [review]
and in markhor.urdf.xacro line 106 and 109.

**Review by GLDuval — APPROVED** [review]

**Review by saxtot — APPROVED** [review]

**Pull request merged**